### PR TITLE
Explicit mention of parameter value for `unite_imu_method`

### PR DESCRIPTION
**Pull request by tsadarsh**
There is confusion regarding what value needs to be passed to `unite_imu_method`. The `ros2-development` branch expects a different value for the `unite_imu_method` parameter as compared to the `ros2-legacy` and `ros1-legacy` branches. I have added this detail in the README. 

Thanks to @Nir-Az for pointing this out in [this](https://github.com/IntelRealSense/realsense-ros/issues/2542#issuecomment-1318831114) issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -232,8 +232,8 @@ For setting a new value for a parameter use `ros2 param set <node> <parameter_na
 
 - **unite_imu_method**: The D435i and T265 cameras have built in IMU components which produce 2 unrelated streams: *gyro* - which shows angular velocity and *accel* which shows linear acceleration. Each with it's own frequency. By default, 2 corresponding topics are available, each with only the relevant fields of the message sensor_msgs::Imu are filled out.
 Setting *unite_imu_method* creates a new topic, *imu*, that replaces the default *gyro* and *accel* topics. The *imu* topic is published at the rate of the gyro. All the fields of the Imu message under the *imu* topic are filled out.
-   - **linear_interpolation**: Every gyro message is attached by the an accel message interpolated to the gyro's timestamp.
-   - **copy**: Every gyro message is attached by the last accel message.
+   - **linear_interpolation**: Every gyro message is attached by the an accel message interpolated to the gyro's timestamp when `unite_imu_method:=2`.
```
Comment (Nir-Az) [inline README.md]: How about this change:

```
unite_imu_method parameter supported values are [0-2] meaning:  [0 -> None, 1 -> Copy, 2 -> Linear_ interpolation]
when:
- **linear_interpolation**: Every gyro message ..
```

Comment (tsadarsh) [reply README.md]: Yes this change is fine

Comment (Nir-Az) [reply README.md]: Do you wish to push the update commit or should I?

**Comment by tsadarsh** [comment]
@Nir-Az I have made the changes and committed. Please have a look into it.

**Comment by Nir-Az** [comment]
Thanks!

**Pull request merged**